examples/walkthrough/k8s201.md

```diff
@@ -12,29 +12,79 @@ certainly want the docs that go with that version.</h1>
 <!-- END STRIP_FOR_RELEASE -->
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
-# Kubernetes 201 - Labels, Replication Controllers, Services and  Health Checking
+# Kubernetes 201 - Labels, Replication Controllers, Services & Health Checking
 
-### Overview
-When we had just left off in the [previous episode](README.md) we had learned about pods, multiple containers and volumes.
-We'll now cover some slightly more advanced topics in Kubernetes, related to application productionization, deployment and
+If you went through [Kubernetes 101](README.md), you learned about kubectl, pods, volumes, and multiple containers.
+For Kubernetes 201, we will pick up where 101 left off and cover some slightly more advanced topics in Kubernetes, related to application productionization, deployment and
 scaling.
 
-### Labels
-Having already learned about Pods and how to create them, you may be struck by an urge to create many, many pods.  Please do!  But eventually you will need a system to organize these pods into groups.  The system for achieving this in Kubernetes is Labels.  Labels are key-value pairs that are attached to each object in Kubernetes.  Label selectors can be passed along with a RESTful ```list``` request to the apiserver to retrieve a list of objects which match that label selector.  For example:
+In order for the kubectl usage examples to work, make sure you have an examples directory locally, either from [a release](https://github.com/GoogleCloudPlatform/kubernetes/releases) or [the source](https://github.com/GoogleCloudPlatform/kubernetes).
+
+Table of Contents
+- [Labels](#labels)
+- [Replication Controllers](#replication-controllers)
+  - [Replication Controller Management](#replication-controller-management)
+- [Services](#services)
+  - [Service Management](#service-management)
+- [Health Checking](#health-checking)
+  - [Process Health Checking](#process-health-checking)
+  - [Application Health Checking](#application-health-checking)
+- [What's Next?](#whats-next)
+
+
+## Labels
+
+Having already learned about Pods and how to create them, you may be struck by an urge to create many, many pods.  Please do!  But eventually you will need a system to organize these pods into groups.  The system for achieving this in Kubernetes is Labels.  Labels are key-value pairs that are attached to each object in Kubernetes.  Label selectors can be passed along with a RESTful ```list``` request to the apiserver to retrieve a list of objects which match that label selector.
+
+To add a label, add a labels section under metadata in the pod definition:
+
+```
+  labels:
+    name: nginx
+```
+
+For example, here is the nginx pod definition with labels ([pod-nginx-with-label.yaml](pod-nginx-with-label.yaml)):
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+  labels:
+    name: nginx
```

Comment:
Please don't use labels with the key "name". People find that confusing. I suggest "app", "component", or somesuch.

See #7257 for more best practices for examples.